sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -11,14 +11,7 @@
 #include "azure/core/platform.hpp"
 
 #include <cstring>
```

Comment:
With this header as is, with all the function bodies moved to cpp, I don't think `#niclude <cstring>` (not `<string>`) is needed anymore, so should be moved to cpp file. `<cstring>` is for `std::snprintf()`, I think.

Reply:
`<cstdio>` is for `std::snprintf()`. `<cstring>` is for `std::memcpy`.

Reply:
Still, if we keep it as is, with just the function signatures, I think, memcpy is not used in the header, just in the cpp.